semgrep-core/tests/php/dots_args.php

```diff
@@ -1,8 +1,8 @@
 <?php
-
 function foo() {
   //ERROR:
     foo(1,2,3,4,5);
   //ERROR:
     foo(5);
 }
+?>
```

Comment:
you actually don't need that in practice.
Most php tools will accept code without the end ?>

Reply:
good to know 👍